### ISSUE TITLE
你好，我运行COCO数据集的时候出现， ERROR - The testing results of the whole dataset is empty.

### ISSUE THREAD

**Issue by wuwu362** [wangchen1801/FPD#22]
2024-12-24 12:01:55,892 - mmfewshot - INFO - workflow: [('train', 1)], max: 100 iters
2024-12-24 12:01:55,892 - mmfewshot - INFO - Checkpoints will be saved to /root/autodl-tmp/project/FPD/work_dirs/fpd_r101_c4_2xb4_coco_base-training by HardDiskBackend.
2024-12-24 12:03:39,400 - mmcv - INFO - Reducer buckets have been rebuilt in this iteration.
2024-12-24 12:04:10,418 - mmfewshot - INFO - Iter [50/100]	lr: 1.998e-04, eta: 0:00:46, time: 0.929, data_time: 0.078, memory: 17742, loss_cls: 2.4474, loss_bbox: 0.0752, acc: 67.8379, loss_meta_cls: 0.0694, meta_acc: 1.4333, loss_rpn_cls: 0.5054, loss_rpn_bbox: 0.1220, loss: 3.2193
2024-12-24 12:04:42,144 - mmfewshot - INFO - Saving checkpoint at 100 iterations
2024-12-24 12:04:46,395 - mmfewshot - INFO - Iter [100/100]	lr: 3.996e-04, eta: 0:00:00, time: 0.721, data_time: 0.069, memory: 17933, loss_cls: 0.6417, loss_bbox: 0.1407, acc: 92.1602, loss_meta_cls: 0.0694, meta_acc: 1.6333, loss_rpn_cls: 0.2296, loss_rpn_bbox: 0.1142, loss: 1.1956
2024-12-24 12:04:46,405 - mmdet - INFO - starting model initialization...
2024-12-24 12:06:06,002 - mmdet - INFO - model initialization done.
2024-12-24 12:12:04,730 - mmfewshot - INFO - Evaluating bbox...
2024-12-24 12:12:04,732 - mmfewshot - ERROR - The testing results of the whole dataset is empty.
2024-12-24 12:12:04,843 - mmfewshot - INFO - Iter(val) [2500]	
/root/miniconda3/envs/fpd/lib/python3.8/site-packages/torch/distributed/launch.py:178: FutureWarning: The module torch.distributed.launch is deprecated
and will be removed in future. Use torchrun.
Note that --use_env is set by default in torchrun.
If your script expects `--local_rank` argument to be set, please
change it to read from `os.environ['LOCAL_RANK']` instead. See 
https://pytorch.org/docs/stable/distributed.html#launch-utility for 
further instructions

  warnings.warn(
